Product.cs

```diff
@@ -32,7 +32,12 @@ public Product(string name, int upc, decimal price)
         }
         public string DisplayProductPrice()
         {
-            return $"Tax = {PriceCalculations.TaxPercentage} %, discount = {PriceCalculations.DiscountPercentage} % Tax amount = ${tax}; Discount amount = ${discount} Price before = ${BasicPrice}, price after = ${TotalPrice}";
+            return $"Tax = {PriceCalculations.TaxPercentage} %, " +
```

Comment:
we always prefer to separate data from behavior, let's discuss that in the sync meeting